scripts/tidy.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/bash
```

Comment:
can we make it a all-in-one checker for clang-format and tidy?

Reply:
@Menooker  OK. I will extend the feature of the script